types/math/dec.go

```diff
@@ -0,0 +1,126 @@
+package math
+
+import (
+	"github.com/cockroachdb/apd/v2"
+	"github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+type Dec struct {
+	dec apd.Decimal
+}
+
+var dec128Context = apd.Context{
+	Precision:   34,
```

Comment:
Why 34. Can we add a comment to it (eg link to a related discussion)?

Reply:
Done!